tests/constants.py

```diff
@@ -0,0 +1 @@
+TF_IRIS_DATASET_S3_PATH = "s3://smdebug-testing/datasets/"
```

Comment:
this path is not specific to Iris for TF, so maybe drop the prefix TF_IRIS

Reply:
Yes